feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/balance/list/BalanceListViewModel.kt

```diff
@@ -89,9 +91,9 @@ class BalanceListViewModel(
         walletInteractor::syncAllNfts
     )
 
-    private val assetsFlow = walletInteractor.assetsFlow()
+    private val externalBalancesFlow = externalBalancesInteractor.observeExternalBalances()
```

Comment:
We also do that in `AssetListMixin`. Maybe worth passing this flow to the factory to avoid making two requests for the same data?